packaging/aws-cloudformation/firecamp.template

```diff
@@ -237,6 +237,12 @@
                 "m4.2xlarge",
                 "m4.4xlarge",
                 "m4.10xlarge",
+                "m5.large",
+                "m5.xlarge",
+                "m5.2xlarge",
+                "m5.4xlarge",
+                "m5.12xlarge",
+                "m5.24xlarge",
```

Comment:
any reason the m5.12xlarge and 24xlarge are not added in firecamp-master.template?

Reply:
yeah, I decided nobody would like to run a bastion host on such a huge instances :)

Reply:
make sense. thanks!